src/appMain/sdl_preloaded_pt.json

```diff
@@ -634,6 +634,7 @@
                             "vin",
                             "wiperStatus",
                             "stabilityControlsStatus",
+                            "seatOccupancy",
```

Comment:
Would it be better to to include this in `DrivingCharacteristics-3`? I don't have a strong opinion on the matter, but `beltStatus` is included in that group, and that has a fairly similar function to this field.

Reply:
@jacobkeeler Done in 12201aa